packages/blade/src/components/Tooltip/TooltipInteractiveWrapper.web.tsx

```diff
@@ -1,22 +1,26 @@
-import styled from 'styled-components';
+import React from 'react';
 import { useTooltipContext } from './TooltipContext';
+import type { BaseBoxProps } from '~components/Box/BaseBox';
 import BaseBox from '~components/Box/BaseBox';
 import { metaAttribute, MetaConstants } from '~utils/metaAttribute';
 
-const TooltipInteractiveWrapper = styled(BaseBox).attrs(() => {
-  return {
-    tabIndex: -1,
-    ...metaAttribute({
-      testID: 'tooltip-interactive-wrapper',
-      name: MetaConstants.TooltipInteractiveWrapper,
-    }),
-  };
-})(() => {
-  useTooltipContext();
+const TooltipInteractiveWrapper = React.forwardRef<HTMLDivElement, Omit<BaseBoxProps, 'as'>>(
+  (props, ref) => {
+    useTooltipContext();
 
-  return {
-    display: 'inline-block',
-  };
-});
+    return (
+      <BaseBox
+        ref={ref}
+        tabIndex={-1}
+        display={props.display || 'inline-block'}
```

Comment:
```suggestion
        display="inline-block"
```

This is not needed right? since we're destructuring props below

Reply:
🤔 right..